src/engraving/rendering/score/slurtielayout.cpp

```diff
@@ -1157,7 +1157,7 @@ Shape SlurTieLayout::getSegmentShape(SlurSegment* slurSeg, Segment* seg, ChordRe
 
     // Remove items that the slur shouldn't try to avoid
     segShape.remove_if([&](ShapeElement& shapeEl) {
-        if (!shapeEl.item() || !shapeEl.item()->parentItem()) {
+        if (!shapeEl.item() || !shapeEl.item()->parentItem() || !shapeEl.item()->visible()) {
```

Comment:
Is it expected that invisible items are added to shapes at all?

Reply:
I think I tried at some point, but it turned out that removing them from shapes altogether doesn't work as there can be situations where you do need them